data/region_border_points.js

```diff
@@ -0,0 +1,149 @@
+var region_border_points = {
```

Comment:
@lynnnakiganda , please remove `is03_1`, `iso3_2` and `Type` columns

Reply:
Okay